test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -26,20 +27,21 @@ class TestFunctional(common_utils.TorchaudioTestCase):
 
     backend = "default"
 
-    def assert_batch_consistency(self, functional, batch, *args, atol=1e-8, rtol=1e-5, seed=42, **kwargs):
-        n = batch.size(0)
-
+    def assert_batch_consistency(self, functional, inputs, atol=1e-8, rtol=1e-5, seed=42):
+        n = inputs[0].size(0)
+        if len(inputs) > 1:
```

Comment:
`if len(inputs) > 1:` is redundant. `range(1, len(inputs))` will handle the case `len(inputs) == 1`.